### DISCUSSION TITLE
Camera image cannot be played

### DISCUSSION THREAD

**Discussion by volkankoc** [bluenviron/mediamtx#2233] [Questions]
This camera tplink tapo c200
config 
  kamera3:
    source: rtsp://admin:admin@192.168.1.66:554/stream1

command line details
?[90m2023/08/22 16:08:35 ?[0m?[32mINF?[0m [path kamera3] [RTSP source] started
?[90m2023/08/22 16:08:35 ?[0m?[32mINF?[0m [path kamera3] [RTSP source] ready: 2 tracks (H264, G711)
?[90m2023/08/22 16:08:35 ?[0m?[32mINF?[0m [HLS] [muxer kamera3] is converting into HLS, 1 track (H264)

player url
http://192.168.1.60:8888/kamera3/stream.m3u8

it is working other cameras. But this camera codec g711.
I don't know why the image is not showing

**Comment by aler9** [comment]
Please open an issue and follow instructions in order to provide data that allows to replicate the issue.
